shared/common-adapters/hoc-timers.js

```diff
@@ -1,53 +1,54 @@
 // @flow
 import * as React from 'react'
 
-export type TimerProps = {|
+type TimerProps = {
```

Comment:
The original mistake here was that `TimerProps` was exact, but we were doing `<T: TimerProps>` below, so the new flow was rightly complaining that all the uses of `HOCTimers` were adding props. So the fix is to make `TimerProps` unexact, which is fine since `PropsWithTimer` combines `P` with `TimerProps` to an exact type.